File: ontrack-model/src/main/java/net/nemerosa/ontrack/model/security/SecurityRole.java
package net.nemerosa.ontrack.model.security;

public enum SecurityRole {

    ADMINISTRATOR("ROLE_ADMIN"),

    USER("ROLE_USER");

    /**
     * Security role name
     */
    private final String roleName;

    SecurityRole(String roleName) {
        this.roleName = roleName;
    }

    public String getRoleName() {
        return roleName;
    }

    public String getRoleAbbreviatedName() {
        return roleName.substring(5);
    }
}

File: ontrack-model/src/main/java/net/nemerosa/ontrack/model/security/SecurityRole.kt
package net.nemerosa.ontrack.model.security

enum class SecurityRole(
        /**
         * Security role name
         */
        val roleName: String
) {

    ADMINISTRATOR("ROLE_ADMIN"),
    USER("ROLE_USER");

    val roleAbbreviatedName: String = roleName.substring(5)

}